### ISSUE TITLE
Filebeat netflow module in a frozen state not processing any events

### ISSUE THREAD

**Issue by Aqualie** [elastic/beats#15423]
Running Filebeat 7.5.1 today I logged into Kibana to check out some of my netflow stats in one of the preconfigured netflow dashboards only to find out that there were 0 results being returned. Switched into Kibana and filtered into the netflow service type and expanded the time-frame to last three days and there are 0 results...

Flipped on a quick pcap on the netflow port and there are hundreds of events being emitted as expected. Looking at all the available filebeat logs and the only thing in all of them "filebeat.log.1 - filebeat.log.7" are:
`2020-01-09T10:13:51.031-0500    INFO    harvester/forwarder.go:52   Input outlet closed`
```
- module: netflow
   log:
     enabled: true
     var:
       netflow_host: x.x.x.x
       netflow_port: 4739
```

**Comment by adriansr** [comment]
I think this could be caused by one of the bugs fixed in https://github.com/elastic/beats/pull/15351

The fix will be available in 7.5.2 soon.

Did the problem happen again soon after restarting the Filebeat? That would be useful. I can share a snapshot version of 7.5.2 with the fix so you can test

**Comment by Aqualie** [comment]
Yes it was fixed after restarting. That'd be great.
